launchable/__main__.py

```diff
@@ -16,11 +18,6 @@
 from .version import __version__
 
 
-class AppBase(object):
```

Comment:
Moved over to `app.py`; I felt a little uneasy importing from `launchable.__main__` for some reasons.